cdn/signed-urls/src/main/java/com/google/cdn/SignedUrls.java

```diff
@@ -75,6 +76,61 @@ public static String getSignature(byte[] privateKey, String input)
   }
   // [END cloudcdn_sign_url]
 
+  // [START cloudcdn_sign_url_prefix]
+  /**
+   * Creates a signed URL with a URL prefix for a Cloud CDN endpoint with the given key. Prefixes
+   * allow access to any URL with the same prefix, and can be useful for granting access broader
+   * content without signing multiple URLs.
+   *
+   * @param requestUrl URL of request
+   * @param urlPrefix URL prefix to sign as a string. urlPrefix must start with either http:// or
+   * https:// and should not include query parameters
+   * @param key url signing key uploaded to the backend service/bucket, as a 16-byte array
+   * @param keyName the name of the signing key added to the back end bucket or service
+   * @param expirationTime the date that the signed URL expires
+   * @return a properly formatted signed URL with URL prefix
+   * @throws InvalidKeyException when there is an error generating the signature for the input key
+   * @throws NoSuchAlgorithmException when HmacSHA1 algorithm is not available in the environment
+   * @throws IllegalArgumentException when urlPrefix string is malformed.
+   */
+  static String signUrlWithPrefix(String requestUrl,
+      String urlPrefix,
+      byte[] key,
+      String keyName,
+      Date expirationTime)
```

Comment:
Kindly move the parameters to a single line and adding line breaks only if they reach the character limit.